beacon-chain/core/state/benchmarks_test.go

```diff
@@ -16,28 +16,6 @@ import (
 
 var runAmount = 25
 
-func TestBenchmarkExecuteStateTransition(t *testing.T) {
```

Comment:
We shouldn't delete this.

CC @0xKiwi to regen the files

Reply:
@terencechain that benchmark is basically duplicated below. Theres the same benchmark with a full block right below it which passes correctly.

Reply:
This is a test though. Below `BenchmarkExecuteStateTransition_FullBlock` is a benchmark which doesn't get run during test

Reply:
Fixing it up, nice catch- the naming is weird